src/index.js

```diff
@@ -43,6 +44,10 @@ const snapshotDiff = (valueA: any, valueB: any, options?: Options): string => {
     difference = stripAnsi(difference);
   }
 
+  if (mergedOptions.stablePatchmarks && !mergedOptions.expand) {
+    difference = difference.replace(/^@@ -[0-9]+,[0-9]+ \+[0-9]+,[0-9]+ @@$/gm, '@@ --- --- @@')
```

Comment:
I don't think the miltiline flag is necessary in the regex

Reply:
It is neccessary. Otherwise, `^` and `$` only match the start and end of the entire string, not every line.

Reply:
Ah, missed that!